packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -103,15 +109,18 @@ export type AllowedActions =
   | TokensControllerGetStateAction
   | NetworkControllerGetNetworkClientByIdAction
   | NetworkControllerGetStateAction
-  | PreferencesControllerGetStateAction;
+  | PreferencesControllerGetStateAction
```

Comment:
Do we still need this one? Is it for retro-compatibility? We don't use it in this PR